front/lib/api/assistant_builder/types.ts

```diff
@@ -0,0 +1,78 @@
+import type {
```

Comment:
Had to move all that stuff here, as the types are needed in a server-side only file